vendor/cmake/FindODBC.cmake

```diff
@@ -0,0 +1,261 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying file Copyright.txt or
+# https://cmake.org/licensing for details.
+
+# From: GDAL
+
+#[=======================================================================[.rst:
+FindODBC
```

Comment:
Does this differ from https://cmake.org/cmake/help/latest/module/FindODBC.html ? At first glance it looks very similar

Reply:
Good spotting — GDALs targetting older CMake, so we can probably just use upstreams. Will check/try